channels/rdpdr/server/rdpdr_main.c

```diff
@@ -71,6 +118,7 @@ static int rdpdr_server_receive_client_name_request(RdpdrServerContext* context,
 {
 	UINT32 UnicodeFlag;
 	UINT32 ComputerNameLen;
+
```

Comment:
Not related to your changes but the available remaining size should be checked before reading